lib/internal/vm.js

```diff
@@ -117,6 +155,19 @@ function internalCompileFunction(
   return result;
 }
 
+/**
+ * Creates a contextify script.
+ * @param {string} code - The code of the script.
+ * @param {string} filename - The filename of the script.
+ * @param {number} lineOffset - The line offset of the script.
+ * @param {number} columnOffset - The column offset of the script.
+ * @param {Buffer} cachedData - The cached data of the script.
+ * @param {boolean} produceCachedData - Indicates whether to produce cached data.
+ * @param {object} parsingContext - The parsing context of the script.
+ * @param {number} hostDefinedOptionId - The host-defined option ID.
+ * @param {boolean} importModuleDynamically - Indicates whether to import modules dynamically.
```

Comment:
`importModuleDynamically` is a function to use for dynamically importing modules. 

```suggestion
 * @param {import('internal/modules/esm/utils').ImportModuleDynamicallyCallback} importModuleDynamically - Indicates whether to import modules dynamically.
```